plugins/modules/remote_management/redfish/ilo_redfish_info.py

```diff
@@ -34,136 +35,214 @@
+        command:
+            description: Returns the output msg and whether the function executed successfully.
             type: dict
             contains:
                 ret:
-                    description: Check variable to see if the information was successfully retrieved.
+                    description: Return True/False based on whether the information was retrived succesfully.
                     type: bool
                 msg:
-                    description: Information of all active iLO sessions.
-                    type: list
-                    elements: dict
-                    contains:
-                        Description:
-                            description: Provides a description of the resource.
-                            type: str
-                        Id:
-                            description: The sessionId.
-                            type: str
-                        Name:
-                            description: The name of the resource.
-                            type: str
-                        UserName:
-                            description: Name to use to log in to the management processor.
-                            type: str
+                    description: Information of all retrived details.
+                    type: dict
     returned: always
-'''
+"""
 
-CATEGORY_COMMANDS_ALL = {
-    "Sessions": ["GetiLOSessions"]
-}
+CATEGORY_COMMANDS_ALL = {"Sessions": ["GetiLOSessions"],
+                         "Systems": ["GetServiceBiosAttributes", "GetBootSettings", "GetPhysicalDrives",
+                                     "GetLogicalDrives", "GetLogicalDrivesWithArrayControllers", "GetServerPostState", "GetNvmeDrives"],
+                         "Manager": ["GetSNMPv3Users", "GetSNMPAlertDestinations"]}
 
-CATEGORY_COMMANDS_DEFAULT = {
-    "Sessions": "GetiLOSessions"
-}
+CATEGORY_COMMANDS_DEFAULT = {"Sessions": "GetiLOSessions"}
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible_collections.community.general.plugins.module_utils.ilo_redfish_utils import iLORedfishUtils
+from ansible.module_utils._text import to_native
+from ansible_collections.community.general.plugins.module_utils.ilo_redfish_utils import (
+    iLORedfishUtils, iLO_certificate_login
+)
 
 
 def main():
     result = {}
     category_list = []
     module = AnsibleModule(
         argument_spec=dict(
-            category=dict(required=True, type='list', elements='str'),
-            command=dict(required=True, type='list', elements='str'),
+            category=dict(required=True, type="list", elements="str"),
+            command=dict(required=True, type="list", elements="str"),
             baseuri=dict(required=True),
             username=dict(),
             password=dict(no_log=True),
             auth_token=dict(no_log=True),
-            timeout=dict(type='int', default=10)
+            timeout=dict(type="int", default=60),
```

Comment:
This is a breaking change. 
Please create separate PRs for new modules and for changes in existing ones, and for this case, you must give users a warning before changing the default value.

Reply:
This is a show-stopper, the PR will not be merged without addressing this.

Deprecating a default value is not a short process. See https://github.com/russoz-ansible/ansible-contrib-unofficial/blob/main/deprecations.md#deprecating-a-parameter-default-value for details.

Reply:
Will work on this

Reply:
Hi, I've asked this above as well, but just adding it here as well, since we have the same module_utils file i.e. ilo_redfish_utils.py for all three module ilo_redfish_info, ilo_redfish_config and ilo_redfish_command, how should we add the ilo_redfish_utils.py while creating separate PRs.